### DISCUSSION TITLE
咸鱼上有人卖

### DISCUSSION THREAD

**Discussion by raven-114** [mikumifa/biliTickerBuy#297] [聊天专区]
![ee2e56fdb6742afff67852a6335b9eb](https://github.com/mikumifa/biliTickerBuy/assets/174227975/249ccef4-839a-44e4-83e3-2be19cc1055a)
如图，这人虽然标注了软件免费但是教程天价（个人认为这个软件大大做的很简洁容易操作，这种教程不值50）

**Comment by mikumifa** [comment]
是的呢

**Comment by ChrisZ-123** [comment]
我靠，早知道先搜下github了，白交50学费
